modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -40,324 +40,467 @@
         "temperature is between 10 and 20 degrees. Also used for masking out line "
         "topologies like solar wind & closed lines."
     };
-    constexpr openspace::properties::Property::PropertyInfo MaskingMinInfo = {
-        "maskingMinLimit",
+    constexpr openspace::properties::Property::PropertyInfo MaskingMinMaxInfo = {
+        "MaskingMinLimit",
         "Lower Limit",
-        "Lower limit of the valid masking range"
-    };
-    constexpr openspace::properties::Property::PropertyInfo MaskingMaxInfo = {
-        "maskingMaxLimit",
-        "Upper Limit",
-        "Upper limit of the valid masking range"
+        "Lower and upper limit of the valid masking range"
     };
     constexpr openspace::properties::Property::PropertyInfo MaskingQuantityInfo = {
-        "maskingQuantity",
+        "MaskingQuantity",
         "Quantity used for Masking",
         "Quantity used for masking."
     };
-    constexpr openspace::properties::Property::PropertyInfo OriginButtonInfo = {
-        "focusCameraOnParent",
-        "Focus Camera",
-        "Focus camera on parent."
+    constexpr openspace::properties::Property::PropertyInfo LineWidthInfo = {
+        "LineWidth",
+        "Line Width",
+        "This value specifies the line width of the field lines if the "
+        "selected rendering method includes lines."
     };
     constexpr openspace::properties::Property::PropertyInfo TimeJumpButtonInfo = {
-        "timeJumpToStart",
+        "TimeJumpToStart",
         "Jump to Start Of Sequence",
         "Performs a time jump to the start of the sequence."
     };
 
-    enum class SourceFileType : int {
-        Cdf = 0,
-        Json,
-        Osfls,
-        Invalid
-    };
+    struct [[codegen::Dictionary(RenderableFieldlinesSequence)]] Parameters {
+        enum class SourceFileType {
+            Cdf,
+            Json,
+            Osfls
+        };
+        // Input file type. Should be cdf, json or osfls
+        SourceFileType inputFileType;
 
-    float stringToFloat(const std::string& input, float backupValue = 0.f) {
-        float tmp;
-        try {
-            tmp = std::stof(input);
-        }
-        catch (const std::invalid_argument& ia) {
-            LWARNING(fmt::format(
-                "Invalid argument: {}. '{}' is NOT a valid number", ia.what(), input
-            ));
-            return backupValue;
-        }
-        return tmp;
-    }
+        // Should be path to folder containing the input files
+        std::filesystem::path sourceFolder [[codegen::directory()]];
+
+        // Path to a .txt file containing seed points. Mandatory if CDF as input.
+        // Files need time stamp in file name like so: yyyymmdd_hhmmss.txt
+        std::optional<std::filesystem::path> seedPointDirectory [[codegen::directory()]];
+
+        // Currently supports: batsrus, enlil & pfss
+        std::optional<std::string> simulationModel;
+
+        //sim mod
+        //std::optional<openspace::fls::Model> model;
```

Comment:
Should everyhtiong related to this actually be removed, and not just commented out?

Reply:
If not, please include a comment somewhere to explain why it's still here :)